test/conformance/kuttl/policy-validation/cluster-policy/admission-disabled/README.md

```diff
@@ -0,0 +1,7 @@
+## Description
+
+This test tries to create various policies with `admission` set to `false`.
+
+## Expected Behavior
+
+Policies containing mutation, image verification or generation rules should be rejected.
```

Comment:
`.spec.admission` only enforces validate policies?

Reply:
can other rules be applied in the background ?